plume-models/src/safe_string.rs

```diff
@@ -101,3 +101,17 @@ impl AsRef<str> for SafeString {
         &self.value
     }
 }
+
+use rocket::request::FromFormValue;
+use rocket::http::RawStr;
+
+impl<'v> FromFormValue<'v> for SafeString {
+    type Error = &'v RawStr;
+
+    fn from_form_value(form_value: &'v RawStr) -> Result<SafeString, &'v RawStr> {
+        let val = String::from_form_value(form_value)?;
+        Ok(SafeString {
+          value: val,
+        })
```

Comment:
`SafeString`s are cleaned at creation, you must either call `SafeString::new(&str)` or `ammonia::clean(&str)` (which is used internally by new) when creating such struct, not built it directly from an unstrusted String.

Reply:
this should be fixed in d62c72d